src/components/Radio/Radio.stories.tsx

```diff
@@ -23,7 +23,6 @@ export default {
     tileDescription: {
       description:
         "Text that can be used to describe the label in more detail. Appears only when `isTile` is set to `true`.",
-      if: { arg: "isTile" },
```

Comment:
Did you intentionally take this out? If we're limiting descriptions to tiled Radio buttons, I like this being a piece.

Reply:
Shoot, I got my wires crossed and misunderstood some feedback. You're right; this should stay in. Must have had Thursday brain 🙃

Reply:
Catching up. Cool we're on the same page.